scene/get-elevation-at-a-point/README.md

```diff
@@ -1,22 +1,22 @@
-# Get elevation at a point
+# Get elevation at point
 
 Get the elevation for a given point on a surface in a scene.
 
-![](GetElevationAtAPoint.png)
+![Image of get elevation at point](GetElevationAtAPoint.png)
 
 ## Use case
 
 Knowing the elevation at a given point in a landscape can aid in navigation, planning and survey in the field.
 
 ## How to use the sample
 
-Click anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.
+Tap anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.
```

Comment:
```suggestion
Click anywhere on the surface to get the elevation at that point. Elevation is reported in meters since the scene view is in WGS84.
```